swingset/src/main/java/com/nqadmin/swingset/utils/SSCommon.java

```diff
@@ -201,6 +204,9 @@ protected class SSRowSetListener implements RowSetListener, Serializable {
 		 */
 		@Override
 		public void cursorMoved(final RowSetEvent event) {
+			if (isAcceptingChanges(rowSet)) {
```

Comment:
I think it would be good to add a little commentary regarding what isAcceptingChanges() is doing and that it's only for CachedRowSet (if I'm following things properly). Otherwise you have to poke around a bit to see what the call is for. Could probably add once in the SSRowSetListener method JavaDoc.

Reply:
Good idea, it took me a while to see what's going on. I started by rewriting the javadoc for `SSDataNavigator.acceptChanges()`. And a bit more elsewhere.